homeassistant/components/rointe/config_flow.py

```diff
@@ -0,0 +1,141 @@
+"""Config flow for Rointe Heaters integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from rointesdk.rointe_api import RointeAPI
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import (
+    CONF_INSTALLATION,
+    CONF_LOCAL_ID,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    DOMAIN,
+    LOGGER,
+)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
```

Comment:
We generally use `cv.string` instead of `str` (except in very unique circumstances): https://github.com/home-assistant/core/blob/96c442349f10e7d9b4823d684c127a3ef5a41d0e/homeassistant/helpers/config_validation.py#L558-L569